ledger-core/application/testwalletapi/statemachine/statemachine.plantuml

```diff
@@ -0,0 +1,19 @@
+@startuml
+Init : migrationDefault
+Init -[#blue]-> migrationDefault
```

Comment:
I'd suggest to use either a dashed line here (.>) instead of color

Reply:
Agree, but that comment related to utility not in a generator. You can leave a comment in issue https://insolar.atlassian.net/browse/PLAT-442